cli/cmd/backup.go

```diff
@@ -67,7 +71,7 @@ func (c *ServicedCli) cmdBackup(ctx *cli.Context)  {
 			return
 		} else {
 			if ! backupSpace.AllowBackup {
-				fmt.Fprintf(os.Stderr, "Unable to backup: estimated space required (%s) exceeds space available (%s) on %s\n", backupSpace.EstimatedString, backupSpace.AvailableString, backupSpace.BackupPath)
+				fmt.Fprintf(os.Stdout, "Unable to backup: estimated space required (%s) exceeds space available (%s) on %s\n", backupSpace.EstimatedString, backupSpace.AvailableString, backupSpace.BackupPath)
```

Comment:
The message is logged in filesystem.go: https://github.com/control-center/serviced/blob/4412af56901bdbdc601df095ada55d35dfc5631f/dao/elasticsearch/filesystem.go#L111-L116. This message is for the user.